README.md

```diff
@@ -71,3 +71,10 @@ It can also re-encrypt them.
 
 ## License
 This application is available as open source under the terms of the [Apache-2.0 License](https://opensource.org/licenses/Apache-2.0).
+
+## Funding
```

Comment:
This project has also been funded by The Calyx Institute.

Maybe

### Calyx Institute
This project is primarily developed and maintained by the [Calyx Institute](https://release.calyxinstitute.org) for usage in [CalyxOS](https://calyxos.org)

### NLnet
< current para >